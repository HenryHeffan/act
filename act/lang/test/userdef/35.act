defproc testme (bool! y);


defproc testme (bool y) {
  prs { 
    true -> y-
  }
}

testme t;
