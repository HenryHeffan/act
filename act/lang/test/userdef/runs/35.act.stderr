ERROR: File `35.act', line 4, col 8
	Name `testme' previously defined as a different process
